R/db-interface.R

```diff
@@ -58,19 +64,32 @@ cdm_copy_to <- nse_function(c(dest, dm, ...,
     abort_no_unique_indexes()
   }
 
+  if (!is.null(table_names)) {
+    if (unique_table_names) {
+      abort_unique_table_names_or_table_names()
+    }
+
+    not_found <- setdiff(names2(table_names), src_tbls(dm))
+    if (has_length(not_found)) {
+      if (unique(not_found)[1] == "") abort_need_named_vec()
```

Comment:
What happens if the second `table_names` is unnamed?

```suggestion
      if (any(not_found == "")) abort_need_named_vec()
```